generator/generator.go

```diff
@@ -389,12 +395,22 @@ func (g *Generator) parseEnum(ts *ast.TypeSpec) (*Enum, error) {
 
 		// Make sure to leave out any empty parts
 		if value != "" {
+			rawName := value
+			valueStr := value
+
 			if strings.Contains(value, `=`) {
 				// Get the value specified and set the data to that value.
 				equalIndex := strings.Index(value, `=`)
 				dataVal := strings.TrimSpace(value[equalIndex+1:])
 				if dataVal != "" {
-					if unsigned {
+					valueStr = dataVal
+					rawName = value[:equalIndex]
+					if enum.Type == "string" {
+						if parsed, err := strconv.ParseInt(dataVal, 10, 64); err == nil {
+							data = parsed
+							valueStr = rawName
```

Comment:
That's a good question... I think for now, keeping backwards compatibility is key, and so how you have it is the better of the two options.  If I get another request for forcing string representation of integer values, then I can always add one more flag to the CLI.  

At some point, I'm going to have to redo the design to have less boolean flags, and more enumerated flags, so that they don't grow so much.

I'm gonna stew on this a bit more, just to make sure I don't think of something else for you to add or check... but right now it looks good to me.

Reply:
Alright, if you don't mind... what about adding an option to specify single or double quotes around a valid integer value in order to allow people to force it to be understood as a string instead of being converted to an int?

I'm fine with using either `'` or `"`, or identifying both.

Reply:
That's a good idea! I'll give it a shot.

I also thought about tags, like on struct fields, but that's more complex and can be done later on top of quotes if needed.